styles/Editor.css

```diff
@@ -958,8 +961,28 @@
 .ProseMirror .table-footnotes-element {
   counter-reset: table_footnote;
 }
+.delete-footnote-dialog button:first-child {
+  width: 96px;
+}
+.delete-footnote-dialog button:not(:first-child) {
+  width: 79px;
+}
+.table-footer .delete-icon {
+  display: none;
+  position: absolute;
+  right: -20px;
+  top: -13px;
+  cursor: pointer;
+  z-index: 1;
+}
+.table-footer .delete-icon:hover svg path {
+  fill: #353535;
+}
+.ProseMirror .table-footer .table-footnotes-element > div {
```

Comment:
I think you don't need that extra `.table-footer`, right?

Reply:
Let's keep  it. It's simply a hierarchy in a selector, which is considered best practice in CSS.

Reply:
I believe that this is not necessarily a best practice in all situations.

Using overly nested selectors can make the CSS harder to understand, debug, and modify. It can also increase the risk of unintended side effects when making changes to class names.

In my opinion, it's important to prioritize clarity, maintainability, and scalability.

Reply:
well it is depending on the HTML structure. I recall using that selector when the class name was "footnotes-element" instead of "table-footnotes-element," distinguishing it from another "footnotes-element" outside the table but sharing the same class name. so it was necessary to add extra selector .